docs/config.md

```diff
@@ -163,33 +156,49 @@ If the sidecar's resource requests are not specified, they are set with the foll
 
 | Request | Quantity |
 |---------|----------|
-| Reports Container CPU | 200m |
-| Reports Container Memory | 384Mi |
+| Reports Container CPU | 500m |
+| Reports Container Memory | 512Mi |
 
 
 ### Resource Requirements
 By default, the operator deploys Cryostat with pre-configured resource requests:
 
 | Request | Quantity |
 |---------|----------|
+| Auth Proxy container CPU | 25m |
+| Auth Proxy container Memory | 64Mi |
 | Cryostat container CPU | 500m |
-| Cryostat container Memory | 256Mi |
+| Cryostat container Memory | 384Mi |
 | JFR Data Source container CPU | 200m |
-| JFR Data Source container Memory | 384Mi |
-| Grafana container CPU | 100m |
-| Grafana container Memory | 120Mi |
-
-Using the Cryostat custom resource, you can define resources requests and/or limits for each of the three containers in Cryostat's main pod:
+| JFR Data Source container Memory | 200Mi |
+| Grafana container CPU | 25m |
+| Grafana container Memory | 80Mi |
+| Database container CPU | 25m |
+| Database container Memory | 64Mi |
+| Storage container CPU | 50m |
+| Storage container Memory | 256Mi |
+
+Using the Cryostat custom resource, you can define resources requests and/or limits for each of the six containers in Cryostat's main pod:
```

Comment:
```suggestion
Using the Cryostat custom resource, you can define resources requests and/or limits for each of the containers in Cryostat's main pod:
```

Just so that we don't have to change it later :D